wagtail/admin/templates/wagtailadmin/pages/workflow_history/detail.html

```diff
@@ -26,7 +26,8 @@ <h2>{% icon "clipboard-list" class_name="initial" %} {{ workflow_state.workflow.
         </p>
 
         <p>
-            {% blocktrans trimmed with workflow_state.get_status_display as status %}Status <span class="status-tag primary"> {{ status }}</span>{% endblocktrans %}
+            {% status classname="primary" as status_tag_variable %}workflow_state.get_status_display{% endstatus %}
```

Comment:
I'm not sure this will work as is. Did you happen to validate this manually?

Reply:
well the error was quite simple that  blocktrans doesn't allow any other block tag causing and error, this approached didn't seems to cause any error in the tests
![image](https://user-images.githubusercontent.com/74553951/207445784-beee23aa-887e-44d2-93c1-4fa1e534e03c.png)
in wagtail.admin.tests.pages.test_workflow_history.TestWorkflowHistoryDetail

Reply:
Ok. 

But it looks like the variable `workflow_state.get_status_display` is not used correctly

Should it be `{{ workflow_state.get_status_display }}` instead?

Reply:
yes !